english/NeverGitEver/NeverGitEver.md

```diff
@@ -0,0 +1 @@
+Not doing git ever again. But if you want to try then click [here] (https://www.udacity.com/course/how-to-use-git-and-github--ud775)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.